commands/channel.go

```diff
@@ -23,12 +23,16 @@ var ChannelCreateCmd = &cobra.Command{
 	RunE: withClient(createChannelCmdF),
 }
 
+// ChannelRenameCmd is the command in mmctl to change name and/or display name of existing channel.
```

Comment:
I would suggest: "ChannelRenameCmd is used to change name and/or display name of an existing channel."